stable/mongodb/README.md

```diff
@@ -112,10 +110,20 @@ $ helm install --name my-release -f values.yaml stable/mongodb
 
 ## Replication
 
-In order to set up a MongoDB replica set with primary, secondary and arbiter nodes you can use the values-production.yaml file.
+You can start the MongoDB chart in replica set mode with the following command:
 
 ```bash
-$ helm install --name my-release -f values-production.yaml stable/mongodb
+$ helm install --name my-release stable/mongodb --set replication.enabled=true
+```
+## Production settings and horizontal scaling
+
+The following repo contains the recommended production settings for MongoDB server in an alternative [values file](values-production.yaml). Please read carefully the comments in the values-production.yaml file to set up your environment appropriately.
+
+To horizontally scale this chart, first download the [values-production.yaml](values-production.yaml) file to your local folder, then scale the secondary nodes stateful set:
+
+```console 
+$ helm install --name my-release -f ./values-production.yaml stable/mongodb
+$ kubectl scale statefulset my-mongodb-secondaries-statefulset --replicas=3 
```

Comment:
I think name of the statefulset in the command is incorrect. Should probably be 

```bash
$ kubectl scale statefulset my-release-mongodb-secondary --replicas=3 
```

Ideally we should be documenting the `helm upgrade` way but I understand that the use of random passwords is the issue here

Reply:
fixed